server/testnet.go

```diff
@@ -0,0 +1,314 @@
+package server
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"cosmossdk.io/math"
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	types2 "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/server/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	srvflags "github.com/cosmos/evm/server/flags"
+	"github.com/spf13/cobra"
+
+	zeta "github.com/zeta-chain/node/app"
+	observertypes "github.com/zeta-chain/node/x/observer/types"
+)
+
+const (
+	DefaultTestnetValidatorTokes = "30000000000000000000000"
+	DefaultDelegatorShares       = "30000000000000000000000.000000000000000"
+)
+
+func TestNetCmd(appCreator types.AppCreator) *cobra.Command {
+	return TestnetCmdWithOptions(appCreator, StartCmdOptions{
+		DBOpener:            openDB,
+		StartCommandHandler: start,
+	})
+}
+
+// TestnetCmdWithOptions creates a command that modifies the local state to create a testnet fork.
+// After running this command, the network can be started with the regular start command.
+func TestnetCmdWithOptions(testnetAppCreator types.AppCreator, opts StartCmdOptions) *cobra.Command {
+	if opts.DBOpener == nil || opts.StartCommandHandler == nil {
+		panic("DBOpener and StartCommandHandler must be provided")
+	}
+
+	cmd := &cobra.Command{
+		Use:   "testnet [newChainID] [operatorAddress]",
```

Comment:
I think the naming (here and throughout the PR) is wrong.

We should call it a devnet. A testnet is a test network that replicate stability of mainnet such as one we have, networks that we create and destroy on demand should be considered a devnet